### PR TITLE
xmloscopy: 0.1.2 -> 0.1.3

### PR DESCRIPTION
**Pull request by grahamc**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/tools/text/xml/xmloscopy/default.nix [hunk 1]
```diff
@@ -48,5 +48,6 @@ stdenv.mkDerivation rec {
     homepage = https://github.com/grahamc/xmloscopy;
     license = stdenv.lib.licenses.mit;
     platforms = stdenv.lib.platforms.all;
+    maintainers = [ stdenv.lib.maintainers.grahamc ];
```
Comment (worldofpeace) [inline pkgs/tools/text/xml/xmloscopy/default.nix]: can we `with stdenv.lib;`? it's kinda pretty that way.

File: pkgs/tools/text/xml/xmloscopy/default.nix [hunk 2]
```diff
@@ -6,7 +6,7 @@ docbook5
 }:
 stdenv.mkDerivation rec {
   name = "xmloscopy-${version}";
```
Comment (worldofpeace) [inline pkgs/tools/text/xml/xmloscopy/default.nix]: pname :sparkles:

**Comment by worldofpeace** [comment]
last nitpick in to use `pname` in repo.

File: pkgs/tools/text/xml/xmloscopy/default.nix [hunk 3]
```diff
@@ -5,8 +5,8 @@ fzf, coreutils, libxml2, libxslt, jing, findutils, gnugrep, gnused,
 docbook5
 }:
 stdenv.mkDerivation rec {
-  name = "xmloscopy-${version}";
-  version = "0.1.2";
+  pname = "xmloscopy";
+  version = "0.1.3";
 
   buildInputs = [
     makeWrapper
```
Comment (jonringer) [inline pkgs/tools/text/xml/xmloscopy/default.nix]: not part of your changes, but I think this is "more correct". Doesn't change closure size though, so I guess it doesn't really matter.
```suggestion
  nativeBuildInputs = [  makeWrapper ];
```

**Review by jonringer — APPROVED** [review]
`nix-review` passes on NixOS
diff LGTM
binary works.

**Pull request merged**